### ISSUE TITLE
new improvements to seo diary

### ISSUE THREAD

**Issue by sendtogeo** [seopanel/SeoDiary#6]
Trying to do some work and was looking at the SEO Diary. After the last two updates it no longer works and missing components...
I was thinking on how I manage client information for their links, logins etc... was going to say add a place to add notes but instead I think I have a much better idea and should be easy for you to integrate.

Possibly you can update the Panel Blog with more functionality to allow posting posts public/private. If private, have dropdown to select user(s) that can view.

So at end you end up with ability to post publicly and also create an internal blog for clients that sees the public and private posts, PLUS only specific user(s) see specific posts. If only the user sees, it performs same function as Diary...

![PastedImage-1](https://user-images.githubusercontent.com/281293/72748922-982bb080-3bde-11ea-8a6e-92262edf817d.png)

ALSO, Blog needs bread crumb added to page tops, categories and sidebar menu.
It would add value if on the Blog home page top you add ability to place content (Above the posts)

Side Note SEO Diary. I think the Diary would serve the best purpose auto-creating entries into diary from the various automated Panel tasks.
EXAMPLE:
Article created/submitted - Diary shows Date, Title, Excerpt, Link
Social Post posted - Diary displays date/time, channel/link, post content
Directory submitted - displays date, directory, status

IN SUMMARY... a mix of ideas but at end basically just looking at how to:
1. Inform clients of daily tasks and actions.
2. Be able to enter and store data on clients links and accounts (user accounts, logins, links for each client.
